aries-site/src/utils/hooks/useDarkMode.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useCallback, useMemo } from 'react';
+import { initialize, useEventListener } from '..';
+
+export const useDarkMode = (
+  initialValue = false,
+  {
+    element,
+    classNameDark,
+    classNameLight,
+    onChange,
+    storageKey = 'darkMode',
+    storageProvider,
+    global,
+  } = {},
+) => {
+  const {
+    usePersistedDarkModeState,
+    getDefaultOnChange,
+    getInitialValue,
+    mediaQueryEventTarget,
+  } = useMemo(() => initialize(storageKey, storageProvider, global), [
+    storageKey,
+    storageProvider,
+    global,
+  ]);
+
+  const [state, setState] = usePersistedDarkModeState(
+    getInitialValue(initialValue),
+  );
+
+  const stateChangeCallback = useMemo(
+    () =>
+      onChange || getDefaultOnChange(element, classNameDark, classNameLight),
```

Comment:
I think `classNameDark` and `classNameLight` can be eliminated from these lines.